### DISCUSSION TITLE
HELP opening YimMenu

### DISCUSSION THREAD

**Discussion by Rushp03** [YimMenu/YimMenu#3512] [General]
So i put the ctrl key to open the menu but it wont open and now it says 'Press VK_Rcontrol to open' but it wont open. Can someone tell me how i can reset the settings. I already uninstalled the menu and GTA. Thank you

**Comment by lonelybud** [comment]
duplicate https://github.com/YimMenu/YimMenu/discussions/3116

**Reply by Rushp03** [reply]
What do you mean ?
